### PR TITLE
Run tests also against upcoming versions of symfony

### PR DESCRIPTION
**Pull request by rosier**
Run tests on travis also against upcoming versions of symfony (beta / rc) and dev-master

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,20 +16,23 @@ matrix:
...
+  - phpenv config-rm xdebug.ini || true
   - composer self-update
-  # Set memory to max (memory fail)
-  - '[[ "$ACTION" == "install" ]] || echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini'
-  # Set stability to dev to allow 4.4dev and 5.0dev
-  - '[[ "$ACTION" == "install" ]] || composer config minimum-stability dev'
-  # Change version of symfony when need
-  - '[[ "$ACTION" == "install" ]] || composer config extra.symfony.require $SYMFONY'
+  # Set memory to unlimited for composer update
+  - [[ $ACTION == "install" ]] || echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
```
Comment (stof) [inline .travis.yml:31]: you could simplify the CI config by using the [`COMPOSER_MEMORY_LIMIT` env variable](https://getcomposer.org/doc/03-cli.md#composer-memory-limit) instead.

**Comment by rosier** [comment]
I did add a few test jobs as demo. Let me know which of them you like to keep.

File: .travis.yml (hunk 1 above)
Comment (rosier) [reply .travis.yml]: I did add the COMPOSER_MEMORY_LIMIT env variable

**Comment by javiereguiluz** [comment]
Thank you @rosier.

**Pull request merged**